canvas_modules/common-canvas/src/common-properties/actions/button/index.js

```diff
@@ -15,4 +15,4 @@
  */
 
 import ButtonAction from "./button.jsx";
-module.exports = ButtonAction;
+export default ButtonAction;
```

Comment:
Changing module.exports to export default is required for tree shaking to work correctly.